### ISSUE TITLE
Release 2.0.1.0

### ISSUE THREAD

**Issue by lukas-vlcek** [aiven/prometheus-exporter-plugin-for-opensearch#66]
OpenSearch 2.0.1 has been released, see [release notes](https://github.com/opensearch-project/opensearch-build/blob/main/release-notes/opensearch-release-notes-2.0.1.md).
Official OpenSearch release ticket: https://github.com/opensearch-project/opensearch-build/issues/2165

Checklist:

- [ ] [Compatibility Matrix](https://github.com/aiven/prometheus-exporter-plugin-for-opensearch#compatibility-matrix) is updated
- [ ] Release 2.0.1.0 tag is present in the repo
- [ ] Release artifacts are available and downloadable
- [ ] Release notes published
- [ ] Forum announcement published

**Comment by lukas-vlcek** [comment]
We shall be able to use https://github.com/aiven/prometheus-exporter-plugin-for-opensearch/pull/65

**Comment by Maglore** [comment]
Hi @lukas-vlcek,

any updates on the new version?
should i create a PR for the version bump? mentioned in #65 ?

**Comment by lukas-vlcek** [comment]
@Maglore Yes, if you want to take it, please go ahead. I am happy to review. Something like this https://github.com/aiven/prometheus-exporter-plugin-for-opensearch/pull/31 and we can follow-up with the rest of the steps to finish the reelase.

I was trying to get https://github.com/aiven/prometheus-exporter-plugin-for-opensearch/issues/64 into the release but in the end it was closed. There is nothing we are waiting for atm.

**Comment by Maglore** [comment]
Hi @lukas-vlcek,
any updates here?
Is there something missing for the release?

**Comment by reta** [comment]
hey @Maglore, apologies for delay, @lukas-vlcek is on vacation, looking into PR now ... thank you!

**Comment by Maglore** [comment]
Hi @reta,
i think for the 2.0.1.0 release we need to update the readme which is done in #65 
and it would be nice to have an artifact, and release before we bump the compatible version again with: #76 
Regards

**Comment by lukas-vlcek** [comment]
As @reta pointed out, I was on vacation.
I am back now and I will jump on this... we need to improve our release process so that it is more automated and more people can run it.

**Comment by reta** [comment]
Compatibility matrix updated https://github.com/aiven/prometheus-exporter-plugin-for-opensearch/pull/65

**Comment by rursprung** [comment]
@reta: as you've merged the PRs and seem to be about to merge #76: is there anything missing except for the actual release of 2.0.1.0? i guess that should be done before you merge #76 (and then release that, subsequently)?

as you already merged #65 w/o the actual release the README now contains a link which isn't working yet (and if people blindly copy & paste it they will run into issues)

**Comment by reta** [comment]
> @reta: as you've merged the PRs and seem to be about to merge #76: is there anything missing except for the actual release of 2.0.1.0? i guess that should be done before you merge #76 (and then release that, subsequently)?

Correct

> as you already merged https://github.com/aiven/prometheus-exporter-plugin-for-opensearch/pull/65 w/o the actual release the README now contains a link which isn't working yet (and if people blindly copy & paste it they will run into issues)

Should be available shortly

**Comment by reta** [comment]
```
opensearch-2.0.1$ ./bin/opensearch-plugin install https://github.com/aiven/prometheus-exporter-plugin-for-opensearch/releases/download/2.0.1.0/prometheus-exporter-2.0.1.0.zip
-> Installing https://github.com/aiven/prometheus-exporter-plugin-for-opensearch/releases/download/2.0.1.0/prometheus-exporter-2.0.1.0.zip
-> Downloading https://github.com/aiven/prometheus-exporter-plugin-for-opensearch/releases/download/2.0.1.0/prometheus-exporter-2.0.1.0.zip
[=================================================] 100%   
@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
@     WARNING: plugin requires additional permissions     @
@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
* java.lang.RuntimePermission accessClassInPackage.sun.misc
* java.lang.RuntimePermission accessDeclaredMembers
* java.lang.reflect.ReflectPermission suppressAccessChecks
See http://docs.oracle.com/javase/8/docs/technotes/guides/security/permissions.html
for descriptions of what these permissions allow and the associated risks.

Continue with installation? [y/N]y
-> Installed prometheus-exporter with folder name prometheus-exporter
```
